src/locales/en.json

```
@@ -344,6 +344,26 @@
         "Atomic": "Bug corrected in the periodic table",
         "Gutter": "Gutter corrected in the python application"
       }
+      "1-0-1": {
+        "XNT" : "XNT Cyclic (Multiple presses of XNT change the character inserted between X, N, T and θ)",
+        "NewPol" : "Changed font",
+        "TxtItal" : "Italicized text in some places such as function definitions in the functions application and texts and comments in the Python editor",
+        "TxtTool" : "Larger text in toolboxes with automatic scrolling when text is too long",
```

Can you rename it to `ToolboxFontSize` ?
```suggestion
        "TxtTool" : "Larger text in toolboxes with automatic lateral scrolling when text is too long",
```